framework/src/materials/MaterialPropertyInterface.C

```diff
@@ -88,6 +88,31 @@ MaterialPropertyInterface::defaultMaterialProperty(const std::string & name)
   return nullptr;
 }
 
+const MaterialProperty<Real> *
+MaterialPropertyInterface::zeroMaterialPropertyDot(const std::string & name)
```

Comment:
This should not have a `Dot` in its name. It is more general than just for dot properties. I'm already doing a similar thing for the dervative material properties and might just used this function instead.